templates/partials/greeting.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if recipientName ~}}
```

Comment:
I didn't know handlebars had this `~` feature to ignore [whitespaces](https://handlebarsjs.com/guide/expressions.html#whitespace-control), this is awesome!